### PR TITLE
feat: Local/external signer choice

### PR DESCRIPTION
**Pull request by hudem1**
Allows user to choose between local signer or external signer by adding either the flag `--local-signer` or `--external-signer`

Note: Sometimes i used actual (read only) calls to rpc (with valid/working url in .env), and in my last tests i used httptest for mocking easily returned values allowing me to test edge cases. Idk if it's okay to have both approaches or if we wanna keep only one. I guess whatever we choose, we can update the tests in another PR as this one is a bit urgent i think.

### CODE REVIEW

File: attest.go [hunk 1]
```diff
@@ -24,16 +24,26 @@ func Attest(config Config) error {
...
+	} else {
+		externalSigner, err := NewExternalSigner(provider, config.AccountData.OperationalAddress, config.ExternalSignerUrl)
+		if err != nil {
+			return err
+		}
+		account = &externalSigner
 	}
 
-	dispatcher := NewEventDispatcher[*ValidatorAccount, *utils.ZapLogger]()
+	dispatcher := NewEventDispatcher[Accounter, *utils.ZapLogger]()
```
[the remark this answers is not in the document]
Comment (hudem1) [reply attest.go]: reap ? what do you mean ?
